zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1968,19 +1968,17 @@ components:
     DecisionDefinitionFilterRequest:
       type: object
       properties:
-        id:
-          type: string
-        key:
+        decisionKey:
```

Comment:
🔨  We have been discussing over past definition meetings, once this is part of decision, the prefix decision before the key it is not necessary.

Does anything changed over here @tmetzke ?

Reply:
for reference this are the renamed fields
* `decisionId` to `dmnDecisionId`
* `name` to `dmnDecisionName`
* `decisionRequirementsId` to `dmnDecisionRequirementsId`
* `key` to `decisionKey`

Reply:
@tmetzke any thoughts on this? I don't want to go against any previous decision.